### ISSUE TITLE
Remove dependency on ip.finish

### ISSUE THREAD

**Issue by hyder** [oracle-terraform-modules/terraform-oci-oke#154]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

Given #143 was a red herring (as a result of removing the provider.tf and the oci provider subsequently using the oci config), I think we should remove the dependency on ip.finish file on admin host. 

### New or Affected Resource(s)
Remove all occurrences on ip.finish 

### Potential Terraform Configuration

```hcl
# Copy-paste any Terraform configurations for how the requested feature may be used. 
```

**Comment by hyder** [comment]
@redscaresu @styledigger @sauraahu  I'll need your help to test this

**Comment by hyder** [comment]
I've pushed to my fork and tested both with the provider.tf and without (i.e. the oci config will be used). Can you please test and let me know?

git checkout -b hyder-issue-154 master
git pull https://github.com/hyder/terraform-oci-oke.git issue-154

Thanks

**Comment by styledigger** [comment]
@hyder Did test on issue-154, both apply and destroy worked without issues. Kubectl on admin host worked as well.
Test was done with provider.tf and default terraform.tfvars (just changed region to Frankfurt).

**Comment by hyder** [comment]
Merged in in #156
